src/Interpreters/Cache/FileCache_fwd.h

```diff
@@ -8,7 +8,7 @@ static constexpr int FILECACHE_DEFAULT_MAX_FILE_SEGMENT_SIZE = 32 * 1024 * 1024;
 static constexpr int FILECACHE_DEFAULT_FILE_SEGMENT_ALIGNMENT = 4 * 1024 * 1024; /// 4Mi
 static constexpr int FILECACHE_DEFAULT_BACKGROUND_DOWNLOAD_THREADS = 5;
 static constexpr int FILECACHE_DEFAULT_BACKGROUND_DOWNLOAD_QUEUE_SIZE_LIMIT = 5000;
-static constexpr int FILECACHE_DEFAULT_LOAD_METADATA_THREADS = 1;
+static constexpr int FILECACHE_DEFAULT_LOAD_METADATA_THREADS = 16;
```

Comment:
But if cache directory is really small, it will slow down the load. (I checked). Therefore I left it as 1 by default. Do not remember exact numbers now, but probably we can neglect it...